packages/esm-patient-search-app/src/patient-search-result/__tests__/patient-search-result.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import PatientSearchResults from '../patient-search-result.component';
+import { useConfig } from '@openmrs/esm-framework';
+import { mockSearchResults } from './patient-search-mock-resource';
+
+const mockUseConfig = useConfig as jest.Mock;
+
+jest.mock('@openmrs/esm-framework', () => ({
+  ...(jest.requireActual('@openmrs/esm-framework') as any),
+  formatDate: jest.fn(),
+  parseDate: jest.fn(),
+  useVisit: jest.fn(),
+  age: jest.fn(),
+  useConfig: jest.fn(),
+  ExtensionSlot: jest.fn().mockImplementation(() => 'ExtensionSlot Component'),
+}));
```

Comment:
All of these except `age` are in the [default mock file](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-framework/mock.tsx).

Reply:
good to know... we should move the age into the default mock too... I added a TODO and will fix it with any of changes to esm-framework